app/albums/create/page.tsx

```
@@ -17,20 +17,26 @@ export default function CreatePage() {
 	const uid = useSelector((state: RootState) => state.user.user.uid);
 	const router = useRouter();
 
+	if (!uid) {
+		alert("ユーザーIDが取得できませんでした。");
+		return;
+	}
+
 	const onSubmit: SubmitHandler<Inputs> = async (data) => {
 		try {
 			const documentData: any = {
 				coverImg: data.coverImg,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)